radixdlt/babylon-wallet-android
# ABW-1444 - Default deposit guarantee functionality added.

## Description
https://radixdlt.atlassian.net/browse/ABW-1444

### Screenshots (optional)

https://github.com/radixdlt/babylon-wallet-android/assets/108684750/6e2796f4-9fc7-466b-8e71-a164583a6763

### Notes (optional)
The Default deposit guarantee profile change was done already so here all remaining functionality is covered.

Strings are hardcoded until Crowdin PR gets approved and merged -> https://github.com/radixdlt/apps-localization-src/pull/193

Steps to test :
- Go to settings and play with changing the default deposit guarantee (default is 1.0 which is reflects 100% in the transaction)
- Then once applied, trigger any transaction that involves guarantees (easiest to use one of the examples from Sandbox or Gumball Club Candy Machine)
- Observe that default deposit guarantee is reflected on the review screen (instead of 100%)
